stock_delivery_result_import/wizards/stock_delivery_result_import.py

```diff
@@ -60,7 +60,9 @@ def import_stock_delivery_result(self):
                             "tracking_refs": [],
                             "carrier_info": carrier_info,
                         }
-                pick_dict[picking]["tracking_refs"].append(row_dict.get("tracking_ref"))
+                        pick_dict[picking]["tracking_refs"].append(
+                            row_dict.get("tracking_ref")
+                        )
```

Comment:
Why this change?  It will make the code capture only the first tracking ref if there are multiple tracking refs involved.